app/templates/libros/index.hbs

```diff
@@ -37,10 +36,11 @@
     </div>
 
     <hr>
-    
+
     <p>
-      Visitá la sección<Link @href={{href-to "acercade"}} @openOnNewTab={{false}}><strong>Acerca de</strong></Link>
-      para conocer más en detalle de qué se trata esta herramienta y quién la desarrolló.
+     Visitá la sección<Link @href={{href-to "acercade"}} @openOnNewTab={{false}}><strong>Acerca de</strong></Link>
+           para conocer más en detalle de qué se trata esta herramienta y quién la desarrolló.
```

Comment:
@sandrau El problema que hay acá lo tiraste por Slack: habría que buscar una forma de poder interpolar un componente (en nuestro caso Link, que es un componente propio por la integración con Electron) dentro de la traducción del texto.

Al parecer este es un problema no resuelto, de lo que pude encontrar:
- [La propuesta de html-safe](https://stackoverflow.com/questions/40265294/link-interpolation-i18n%E2%80%8C): que entiendo que hiciste que eso sea válido por default, pero que en este caso no nos sirve porque este es un componente de Ember.
- [Acá se propone dividir el html](https://stackoverflow.com/questions/30759393/ember-i18n-dynamic-link-inside-of-translation/30760017#30760017) que entiendo que es lo que intentamos evitar porque es medio engorroso.
- Uno de todos esos links leva a [este issue del 2013](https://github.com/jamesarosen/ember-i18n/issues/79) con cosas turbias (y quizás deprecadas ya).

En vista de que además comentaste que ya estuvieron peleando sin éxito, si no hay solución elegante a este problema, entonces propongo _hacer trampa_ y dejar el link al final:
```suggestion
     {{ t "libros.index.about_description"}}
     <Link @href={{href-to "acercade"}} @openOnNewTab={{false}}><strong>{{ t "libros.index.about"}}</strong></Link>
```

con la traducción
```json
{
...
"about": "Acerca de",
"about_description": "Para conocer más en detalle de qué se trata esta herramienta y quién la desarrolló, visitá la sección"
}
```

Y @asanzo habría que ver de abrazar la idea de evitar tener textos de este estilo. O, mejor dicho, saber que el precio es descomponer el html.

Reply:
Lo habiamos hablado en algun thread canal con @asanzo .. el problema es que habia otros textos, con n links adentro.. por eso medio quedamos con la solucion de separarlo en partes con nombre.. tipo
acercade:
    texto1: blabl
    link1: linkTranslate
    texto2: blabl

y asi..

Reply:
Y no recuerdo si ese en particular, pero probe muchas cosas turbias, una alternativa era volar el Link y tratar de usar un a hrf con estilo y el href-to, que tampoco me anduvo, pero me parecia muy feo no usar el Link que el dia de mañana puedenponerle mas cosas, y va a quedar separado del que esta traducido, mucho lio para poca ganancia